src/views/EditSimple.vue

```diff
@@ -250,6 +251,14 @@ export default {
 			return this.isVisible || this.widgetEventDetailsOpen
 		},
 
+		topActionsClass() {
+			return {
+				'event-popover__top-actions': true,
```

Comment:
You can keep it static `class="event-popover__top-actions"` and keep the directions classes here.